omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -913,581 +1133,1838 @@
+                                                                 answerLabel="7 - Drug Stock out"/>
+                                                            <obs conceptId="90223" answerConceptId="90153"
+                                                                 answerLabel="11 - Alcohol"/>
+                                                            <span id="enable-disable-other-poor-art-adherence-reason">
+                                                                <obs conceptId="90223" answerConceptId="90002"
+                                                                     answerLabel="15 - Other Specify"/>
+                                                            </span>
+                                                            <span id="other-poor-art-adherence-reason-enabled">
+                                                                <obs conceptId="99125"/>
+                                                            </span>
+                                                        </div>
+                                                        <div class="col-md-3">
+                                                            <obs conceptId="90223" answerConceptId="90148"
+                                                                 answerLabel="4 - Felt better"/>
+                                                            <obs conceptId="90223" answerConceptId="90151"
+                                                                 answerLabel="8 - Patient lost/ran out of pills"/>
+                                                            <obs conceptId="90223" answerConceptId="90154"
+                                                                 answerLabel="12 - Depression"/>
+                                                        </div>
+                                                    </div>
                                                 </div>
-                                                    <label>Syphilis Status</label>
-                                                <div class="col-md-2">
-                                                    <obs conceptId="99753" class="long-value-select" answerConceptIds="162848,99756,99755,99754"
-                                                         answerLabels=" 1 - No symptoms, 2 - Symptomatic Non-Reactive, 3 - Symptomatic Reactive, 4 - Symptomatic Reactive and treated" />
+                                                <div class="row">
+                                                    <obsgroup groupingConceptId="165430">
+                                                        <div class="col-md-3">
+                                                            <obs id="90315" conceptId="90315" labelText="Regimen:"
+                                                                 answerConceptIds="99015,99016,99005,99006,164979,99039,99040,164977,99041,99042,99007,99008,99044
```

Comment:
@slubwama see comment on restricting by permissions rather than by role

Reply:
@ssmusoke not possible yet in HTML form entry